eras/allegra/impl/cddl-files/allegra.cddl

```diff
@@ -40,37 +42,41 @@ operational_cert =
   , sigma           : signature
   )
 
-kes_vkey = bytes .size 32
+kes_vkey = bytes32
+
+signature = bytes64
 
-signature = bytes .size 64
+bytes64 = bytes .size 64
 
-protocol_version = (major_protocol_version, uint)
+protocol_version = [major_protocol_version, uint]
```

Comment:
This change from group to array seems significant. However, I don't see any change in the `.hs` that matches this. I wonder where it's coming from?